Libplanet.Tools/Mpt.cs

```diff
@@ -23,6 +23,14 @@ internal enum SchemeType
 
     public class Mpt
     {
+        private const string KeyValueStoreURIExample =
+            "(e.g., <kv-store-type>://<kv-store-path>, rocksdb:///path/to/kv-store)";
```

Comment:
I guess '`<kv-store-type>://<kv-store-path>'` and `'rocksdb:///path/to/kv-store'` aren't same hierarchy(?). 

```suggestion
            "(e.g., rocksdb:///path/to/kv-store)";
```

Reply:
Then, we remove the `<kv-store-type>://<kv-store-path>` and is there no requirements to comment `default` type also?

Reply:
As `<kv-store-type>://<kv-store-path>` is not an *example*, it should be rather like:

```suggestion
            "<kv-store-type>://<kv-store-path> (e.g., rocksdb:///path/to/kv-store)";
```